### ISSUE TITLE
this is just like pixel art

### ISSUE THREAD

**Issue by ghost** [terroo/wallset#29]
![2021-08-31-121454_1024x768_scrot](https://user-images.githubusercontent.com/80429360/131476432-507ef848-5c89-4558-bc65-97f83b238c82.png)
video 720p but this is pixel art this is normal ?

**Comment by TheHighGroundWins** [comment]
yeah I also have  video at 1080p 60fps but the pixels are visible

**Comment by cngeeks** [comment]
that's because this tool just take screenshots of the video and play these screenshots in loop instead videos.

**Comment by duracell80** [comment]
See #44
